packages/itwinui-css/src/tabs/default.scss

```diff
@@ -43,35 +43,39 @@
     }
   }
 
-  @at-root {
-    .iui-horizontal & {
-      .iui-tab.iui-active {
-        border-bottom-color: transparent;
-      }
+  @at-root .iui-horizontal & {
+    .iui-tab.iui-active {
+      border-bottom-color: transparent;
+    }
 
-      :not(:first-child) .iui-tab {
-        border-left: none;
-      }
+    :not(:first-child) .iui-tab {
+      border-left: none;
+    }
 
-      ~ .iui-tabs-content {
-        margin-top: -1px;
-      }
+    ~ .iui-tabs-content {
+      margin-top: -1px;
+    }
+
+    .iui-button {
+      height: 100%;
     }
   }
 
-  @at-root {
-    .iui-vertical & {
-      .iui-tab.iui-active {
-        border-right-color: transparent;
-      }
+  @at-root .iui-vertical & {
+    .iui-tab.iui-active {
+      border-right-color: transparent;
+    }
 
-      :not(:first-child) .iui-tab {
-        border-top: none;
-      }
+    :not(:first-child) .iui-tab {
+      border-top: none;
+    }
 
-      ~ .iui-tabs-content {
-        margin-left: -1px;
-      }
+    ~ .iui-tabs-content {
+      margin-left: -1px;
+    }
+
+    .iui-button {
+      width: 100%;
```

Comment:
Can we undo all the extra changes except for this part? Will make it easier to avoid merge conflicts.

Also maybe we should use `:where` selector to reduce specificity of this override in case the user wants a different width/height on the button.

Reply:
Ok, change made.